Draft.md

```diff
@@ -63,90 +66,92 @@ keywords_en:
+Privilegien und Ungleichheiten in der altertumswissenschaftlichen Forschung werden unbewusst auch in der Forschungssoftware fortgeschrieben: Für archäologische Feldarbeit wird beispielsweise oft Software zur Datenaufnahme eingesetzt, die für die Mehrheit der Mitarbeiter vor Ort aufgrund einer Sprachbarriere nicht verständlich ist. Eine Folge kann sein, dass diese Teammitglieder sich nicht weiter qualifizieren und so anspruchsvollere Aufgaben nicht übernehmen können. Proprietäre Software oder teure Hardware benachteiligt wiederum Forscher, die mit weniger Finanzmitteln auskommen müssen.
 
-In Archäologien, in denen häufig an fremden Kulturen und Kulturhinterlassenschaften geforscht wird, sollten die indigenen und ortsansässigen Bevölkerungsgruppen die Möglichkeit zur Mitbestimmung über die Erforschung bekommen. Um diesem Ideal gerecht zu werden, eignet sich die Orientierung an den CARE-Prinzipien ([https://www.gida-global.org/care](https://www.gida-global.org/care)). Sie wurden von der [Global Indigenous Data Alliance (GIDA)](https://www.gida-global.org) und der [Research Data Alliance (RDA)](https://www.rd-alliance.org) 2018 auf Basis der [UN Declaration on the Rights of Indigenous Peoples (UNDRIP)](https://www.un.org/development/desa/indigenouspeoples/declaration-on-the-rights-of-indigenous-peoples.html) erarbeitet und konzentrieren sich wie die FAIR-Prinzipien explizit auf Forschungsdaten. Ihre vier Grundpfeiler Kollektiver Nutzen (Collective Benefit), Souveränität (Authority to Control), Verantwortung (Responsibility) und Ethik (Ethics) sind unserer Meinung nach ebenso auch für Forschungssoftware relevant.
+n Archäologien, in denen häufig an fremden Kulturen und Kulturhinterlassenschaften geforscht wird, sollten die indigenen und ortsansässigen Bevölkerungsgruppen die Möglichkeit zur Mitbestimmung über die Erforschung bekommen. Um diesem Ideal gerecht zu werden, eignet sich die Orientierung an den CARE-Prinzipien ([https://www.gida-global.org/care](https://www.gida-global.org/care) [23.10.2020]). Sie wurden von der [Global Indigenous Data Alliance (GIDA)](https://www.gida-global.org) (https://www.gida-global.org [23.10.2020]) und der [Research Data Alliance (RDA)](https://www.rd-alliance.org) (https://www.rd-alliance.org [23.10.2020]) 2018 auf Basis der [UN Declaration on the Rights of Indigenous Peoples (UNDRIP)](https://www.un.org/development/desa/indigenouspeoples/declaration-on-the-rights-of-indigenous-peoples.html) (https://www.un.org/development/desa/indigenouspeoples/declaration-on-the-rights-of-indigenous-peoples.html [23.10.2020]) erarbeitet und konzentrieren sich wie die FAIR-Prinzipien explizit auf Forschungsdaten. Ihre vier Grundpfeiler kollektiver Nutzen (Collective Benefit), Souveränität (Authority to Control), Verantwortung (Responsibility) und Ethik (Ethics) sind unserer Meinung nach ebenso für Forschungssoftware relevant.
 
 ## Mögliche Vorgehensweise zur Rezension archäologischer Forschungssoftware
```

Comment:
> Es wird nicht deutlich, ob die zusammenfassenden Eckdaten eher vorn, eher hinten, oder je nach dem zu platzieren sind. Weiter hinten wird letzteres deutlich, vielleicht lässt sich dies hier mit der Formulierung auch klarer darstellen.

Siehe auch #56

Reply:
Auch an diesem Punkt scheint mir unser Text eigentlich eindeutig. Wir sagen Eckdaten seien zu Beginn vorzustellen, dazu halten wir eine Tabelle für sinnvoll, wie jene, die wir hier ans Ende unseres Beitrages gesetzt haben. Aber um solchen Fehllesungen entgegenzuwirken, würde ich denken, dass wir es auch so machen könnten. "Wir schlagen vor, den Lesern dise Eckdaten in einer tabellarischen Übersicht zur Verfügung zu stellen. Einen Vorschlag, welche Inhalte eine solche Tabelle enthalten kann, haben wir an den Ende dieses Beitrags gesetzt. Alle Einträge dort werden ausführlicher im vorhergehenden Text behandelt."

Reply:
Ich glaube die erste Nennung der Eckdaten im ersten Absatz führte zu der Anmerkung

Reply:
Sieht nun so aus: 

> Unserem Verständnis nach sollte eine Rezension archäologischer Software, so wie es eine herkömmliche Besprechung einer wissenschaftlichen Publikation leistet, die Software zunächst kurz mit zusammenfassenden Eckdaten vorstellen. Anschließend sollte der ausführlichere Kontext erläutert werden und eine kritische Beurteilung erfolgen.

> Zu den Eckdaten, welche in einer tabellarischen Übersicht zur Verfügung gestellt werden können, gehören unter anderem 
Angaben zur Version, zu den Entwicklern und der Lizenz. Dies ermöglicht z. B. eine schnelle Übersicht darüber, ob die begutachtete Software mit der eigenen technischen Umgebung kompatibel ist. Einen Vorschlag zu den Inhalte einer solchen Tabelle haben wir an den Ende dieses Beitrags gesetzt. Details zu allen Einträgen  dort werden ausführlicher im vorhergehenden Text behandelt.